File: SenD.client/src/models/Monster.js
export class Monster {
  constructor(data) {
    this.id = data.id
    this.name = data.name
    this.img = data.img
    this.creator = data.creator
    this.description = data.description
    // this.monsterMessageId = data.monsterMessageId
    this.type = data.type
    this.subType = data.subType
    this.power = data.power
    this.toughness = data.power
  }
}

File: SenD/Models/Monster.cs
namespace SenD.Models;

public class Monster : RepoItem<int>
{
  public string Name { get; set; }
  public string Img { get; set; }
  public string Type { get; set; }
  public string SubType { get; set; }
  public int Power { get; set; }
  public int Toughness { get; set; }
  public string Description { get; set; }
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }

}